src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalPrice = days * 40;
+
+  if (days < 3) {
+    return totalPrice;
+  }
+
+  if (days >= 7) {
+    return totalPrice - 50;
+  } else {
+    return totalPrice - 20;
```

Comment:
It would be easier to reuse code if You will move all numbers to variables.

Reply:
create variables for discounts